### PR TITLE
console cleanup

### PR DESCRIPTION
**Pull request by zburke**
I don't remember what sent me off on this quest, but I discovered these
not-quite-right problems with props, throwing warnings in the console,
along the way. Now, the console is clean.

### CODE REVIEW

File: src/settings/StaffSlips/StaffSlipManager.js [hunk 1]
```diff
@@ -53,26 +55,29 @@ class StaffSlipManager extends React.Component {
...
-        entryFormComponent={StaffSlipForm}
-        validate={this.validate}
-        nameKey="name"
-        permissions={{
-          put: 'ui-circulation.settings.staff-slips',
-          post: 'ui-circulation.settings.staff-slips.post',
-          delete: 'ui-circulation.settings.staff-slips.delete'
-        }}
-      />
+      <IntlConsumer>
```
Comment (aditya-matukumalli) [inline src/settings/StaffSlips/StaffSlipManager.js]: Could we just avoid _IntlConsumer_ here altogether by utilizing the render props of _FormattedMessage_. Similar to what we have [here](https://github.com/folio-org/ui-users/blob/master/src/settings/permissions/PermissionSets.js#L65)

Comment (zburke) [reply src/settings/StaffSlips/StaffSlipManager.js]: Technically yes, and I see the appeal of that simplicity. I probably went for the `<IntlConsumer>` approach out of habit from other situations where there were multiple values requiring translation to strings, so getting `intl` is helpful. So ... I dunno. Render props here would be simpler; `<IntlConsumer>` here would be more consistent. 🤷‍♀

Comment (aditya-matukumalli) [reply src/settings/StaffSlips/StaffSlipManager.js]: Ah okay. Either approach is fine I think. If its more consistent across the app, lets go with the current implementation.

Comment (zburke) [reply src/settings/StaffSlips/StaffSlipManager.js]: Eh, I went for your approach favoring simplicity :)

**Pull request merged**